README.md

```diff
@@ -107,6 +107,15 @@ $builder->add('content', PanelType::class, [
 ]);
 ```
 
+#### Troubleshooting block deletion
+For some unknown reason, blocks are not properly deleted on saving the form, probably because no update is detected. To solve this, you need to create an event on `preUpdate` (in your Sonata Admin for example), to be triggered when you save the form.
```

Comment:
Blocks are removed on the `preUpdate` of the parent entity. So if the entity hasn't been updated but only a removal of blocks was done, those blocks will not be removed. It's more of a doctrine behaviour edge case that causes this issue.

Please use better wording, you cannot say it's for _"some unknown reason"_ in an open source project... You're making the bundle sound like not maintain by reliable people.